qiskit/qpy/__init__.py

```diff
@@ -1621,6 +1621,8 @@ class if it's defined in Qiskit. Otherwise it falls back to the custom
 
 this matches the internal C representation of Python's complex type. [#f3]_
 
+Footnotes
+=========
```

Comment:
I'm open to other names if folks prefer, like Citations or References

Reply:
I think we use references in a few other places:
```suggestion
References
==========
```